Makefile

```diff
@@ -14,6 +14,9 @@ EMPTY :=
 SPACE := $(EMPTY) $(EMPTY)
 COMMA_SEPARATED_SOURCES = $(subst $(SPACE),$(COMMA),$(SOURCES))
 
+YARN := yarn --silent
```

Comment:
This will suppress the `yarn foo v1.2.3` header printed by yarn. I think it is okay to use `$(YARN)` when we use it to invoke another node binaries, e.g. eslint/prettier.